src/app/clusters/thread-border-router-management-server/thread-border-router-management-server.cpp

```diff
@@ -0,0 +1,384 @@
+    CHIP_ERROR err = mDelegate->SetPendingDataset(pendingDataset);
+    return StatusIB(err).mStatus;
+}
+
+void AddDatasetResponse(CommandHandlerInterface::HandlerContext & ctx, Status status, Thread::OperationalDataset & dataset)
+{
+    if (status != Status::Success)
+    {
+        ctx.mCommandHandler.AddStatus(ctx.mRequestPath, status);
+        return;
+    }
+    Commands::DatasetResponse::Type response;
+    response.dataset = dataset.AsByteSpan();
+    ctx.mCommandHandler.AddResponse(ctx.mRequestPath, response);
+}
+
+void ServerInstance::InvokeCommand(HandlerContext & ctxt)
+{
+    switch (ctxt.mRequestPath.mCommandId)
+    {
+    case Commands::GetActiveDatasetRequest::Id:
+        HandleCommand<Commands::GetActiveDatasetRequest::DecodableType>(ctxt, [this](HandlerContext & ctx, const auto & req) {
+            Thread::OperationalDataset dataset;
+            Status status = HandleGetActiveDatasetRequest(IsOverCASESession(ctx), dataset);
+            AddDatasetResponse(ctx, status, dataset);
+        });
+        break;
+    case Commands::GetPendingDatasetRequest::Id:
+        HandleCommand<Commands::GetPendingDatasetRequest::DecodableType>(ctxt, [this](HandlerContext & ctx, const auto & req) {
+            Thread::OperationalDataset dataset;
+            Status status = HandleGetPendingDatasetRequest(IsOverCASESession(ctx), dataset);
+            AddDatasetResponse(ctx, status, dataset);
+        });
+        break;
+    case Commands::SetActiveDatasetRequest::Id:
+        HandleCommand<Commands::SetActiveDatasetRequest::DecodableType>(ctxt, [this](HandlerContext & ctx, const auto & req) {
+            if (!mAsyncCommandHandle.Get())
+            {
+                mPath               = ctx.mRequestPath;
+                mAsyncCommandHandle = CommandHandler::Handle(&ctx.mCommandHandler);
+                ctx.mCommandHandler.FlushAcksRightAwayOnSlowCommand();
+                Status status = HandleSetActiveDatasetRequest(IsFailSafeArmed(ctx.mCommandHandler.GetAccessingFabricIndex()), req);
+                if (status != Status::Success)
+                {
+                    OnActivateDatasetComplete(mSetActiveDatasetSequenceNumber,
+                                              ChipError(ChipError::SdkPart::kIMGlobalStatus, to_underlying(status)));
+                }
+            }
+            else
+            {
+                ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::Busy);
+            }
+        });
+        break;
+    case Commands::SetPendingDatasetRequest::Id:
+        HandleCommand<Commands::SetPendingDatasetRequest::DecodableType>(ctxt, [this](HandlerContext & ctx, const auto & req) {
+            ctx.mCommandHandler.AddStatus(ctx.mRequestPath, HandleSetPendingDatasetRequest(req));
+        });
+        break;
+    default:
+        break;
+    }
+}
+
+CHIP_ERROR ServerInstance::ReadFeatureMap(BitFlags<Feature> & featureMap)
+{
+    bool panChangeSupported;
```

Comment:
Every local variable set by reference should be initialized to a default value. Set this to false by default.

Reply:
Done